docs/modular-resources/advanced/_index.md

```diff
@@ -14,22 +14,22 @@ tags:
     "components",
     "services",
   ]
-description: "Extend Viam with modular resources from the Viam registry."
+description: "Some usage may require you to extend the standardized API definitions or deploy custom components using a server on a remote part"
 aliases:
   - "/program/extend/"
 ---
 
-{{% alert title="Tip" color="tip" %}}
+Some usage may require you to extend the standardized API definitions or deploy custom components using a server on a {{< glossary_tooltip term_id="remote" text="remote part" >}}.
 
-[Modular resources](/modular-resources/) provided by custom {{< glossary_tooltip term_id="module" text="modules" >}} are the preferred method of creating custom resource implementations.
+## Custom Components as Remotes
 
-These options are provided for advanced users who are unable to use the modular resource system with their robot deployments.
+[Modular resources](/modular-resources/) running on the [board](/components/board/) directly connected to your components are the preferred way of managing and controlling custom components.
+However, if you are unable to use [modular resources](/modular-resources/) because you have to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/modular-resources/advanced/custom-components-remotes/) of your robot.
 
-{{% /alert %}}
+## New API Subtypes
 
-If you are unable to use [modular resources](/modular-resources/) with your robot, you can explore the options below for advanced methods of extending Viam's support to new {{< glossary_tooltip term_id="resource" text="resources" >}}.
+The [component APIs](/program/apis/#component-apis) and [service APIs](/program/apis/#service-apis) provide a standard interface for controlling common hardware components and higher level functionality.
 
-{{< cards >}}
-{{% card link="/modular-resources/advanced/create-subtype/" %}}
-{{% card link="/modular-resources/advanced/custom-components-remotes/" %}}
-{{< /cards >}}
+If you want to use most of an existing API but need just a few other functions, try using the [`DoCommand`](/program/apis/#docommand) endpoint and [extra parameters](/program/use-extra-params/) to add custom functionality to an existing subtype.
+
+If your resource does not fit into any of the existing {{< glossary_tooltip term_id="component" text="component" >}} or {{< glossary_tooltip term_id="service" text="service" >}} {{< glossary_tooltip term_id="subtype" text="subtypes" >}} or you want to define different methods for the API, you can [define a new resource _subtype_ and an API for that subtype](/modular-resources/advanced/create-subtype/).
```

Comment:
```suggestion
If your resource does not fit into any of the existing {{< glossary_tooltip term_id="component" text="component" >}} or {{< glossary_tooltip term_id="service" text="service" >}} {{< glossary_tooltip term_id="subtype" text="subtypes" >}} or you want to define different methods for the API, you can [define a new resource subtype and an API for that subtype](/modular-resources/advanced/create-subtype/).
```
[nit] This just got used several times before so I don't think italics make sense after the first usage